packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -1,23 +1,35 @@
 <template>
-    <div class="certifaction-verification">
-        <VerificationDemo v-if="demo !== false" @verify-demo="verifyDemo" @dragging-demo-doc="onDraggingDemoDoc"/>
-
-        <div v-if="filteredVerificationItems.length" class="verification-item-list" ref="results">
-            <VerificationItem
-                v-for="verificationItem in filteredVerificationItems"
-                :key="verificationItem.hash"
-                :verificationItem="verificationItem"/>
-        </div>
+    <div class="certifaction-verification"
+         @dragover.prevent="dragOver"
+         @dragleave="dragLeave"
+         @drop="handleDrop">
 
         <VerificationDropBox
-            @files-dropped="verify"
+            :class="{hidden: !dropbox.draggingOver}"
+            :first-verification="filteredVerificationItems.length === 0"
```

Comment:
There is no prop on this component, so this attribute ends up in the html code.

Reply:
Fixed it